edx_rest_framework_extensions/tests/test_utils.py

```diff
@@ -7,20 +7,54 @@
 from django.test import TestCase
 
 from edx_rest_framework_extensions.tests.factories import UserFactory
-from edx_rest_framework_extensions.utils import jwt_decode_handler
+from edx_rest_framework_extensions import utils
 
+INVALID_TOKEN = 'invalid.token'
 
-def generate_jwt_token(payload):
-    """Generate a valid JWT token for authenticated requests."""
-    return jwt.encode(payload, settings.JWT_AUTH['JWT_SECRET_KEY']).decode('utf-8')
+VALID_ISSUERS = [
+    {
+        'ISSUER': 'valid-issuer',
+        'SECRET_KEY': 'valid-secret-key'
+    },
+    {
+        'ISSUER': 'another-valid-issuer',
+        'SECRET_KEY': 'another-valid-secret-key'
+    }
+
+]
+
+INVALID_ISSUERS = [
```

Comment:
Actually, this isn't really needed at all. You can just use a hardcoded value in the test where this is used.